data/successors/successor side missions.txt

```diff
@@ -21,7 +21,7 @@ mission "Successors: Predecessor Research 1"
 	deadline 30
 	to offer
 		has "trusted by the successors"
-		has "flagship attribute: quantum keystone" > 0
+		"flagship attribute: quantum keystone" > 0
```

Comment:
```suggestion
		has "flagship attribute: quantum keystone"
```
Doesn't this do the same thing?

Reply:
Probably, but does `has` check for just one?
I opted to go for `> 0` just in case you have multiple keystones and `has` didn't do what I hoped.

Reply:
`has` checks that it's nonzero, and it can't be negative, so it must be positive.

Doesn't really matter.

Reply:
I believe I'll leave it like this, like the two files changed above it, unless you've got strong preferences the other way.